hdkeychain/extendedkey_test.go

```diff
@@ -1013,3 +1014,38 @@ func TestZero(t *testing.T) {
 		}
 	}
 }
+
+func TestExtendedKey_MaximumDepth(t *testing.T) {
```

Comment:
Well none of the rest of the code base uses them, except for examples where it is required to make them show up properly on godoc, so I'd like to keep the consistency regardless.